EXPLAINER.md

```diff
@@ -1,118 +1,108 @@
 # Distributed Tracing Working Group
 
-## What is it?
+## Introduction
+The Distributed Tracing Working Group currently drives the standardization of two specifications:
 
-We propose a set of HTTP headers which propagate a distributed trace and associated data, even when a request and its associated trace is served by multiple tracing providers.
+- W3C TraceContext
+- W3C Baggage
```

Comment:
Suggestion: link the above items to the respective specs and delete everything below.

Reply:
Sure, I will add links to the respective specs, however my understanding is that this "explainer" file/content is still needed during various W3C reviews (https://tag.w3.org/explainers/). For example, I have seen various reviews asking for a link to the explainer file. Would be happy to update this content though (to make it crisper or to elaborate where needed).